.github/workflows/scheduled.yml

```
@@ -0,0 +1,60 @@
+name: Scheduled jobs
+
+on:
+  schedule:
+   - cron: "15 5 * * 1"
```

If you want to test running the workflow, you can temporarily change the event that triggers running the workflow so it runs when pushing the commit:

```suggestion
on: push
```

After testing, revert back to the scheduled event. :nerd_face: